### PR TITLE
Add photos when adding a new place to map

### PR DESCRIPTION
**Pull request by sophia-qin**
Allows users to add a photo in addition to name, x_coord, and y_coord when adding a place.

### CODE REVIEW

File: frontend/src/addPlace.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return (
+      <div>
+        <h1>Add Place</h1>
+        <textarea
+          placeholder={"Enter a name"}
+          value={this.state.name}
+          onChange={e => {
+            this.setState({ name: e.target.value });
+          }}
+        ></textarea>
```
Comment (tylerhou) [inline frontend/src/addPlace.js]: Nit: you should self close tags that don't have children.

File: frontend/src/addPlace.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            const formData = new FormData();
+            formData.append("name", this.state.name);
+            formData.append("x_coord", this.state.x_coord);
+            formData.append("y_coord", this.state.y_coord);
+            formData.append("image", this.state.image);
+            fetch("/api/place", {
+              method: "POST",
+              credentials: "same-origin",
+              body: formData
+            }).then(() => window.location.reload());
```
Comment (tylerhou) [inline frontend/src/addPlace.js]: Why do we want to reload here?

Comment (sophia-qin) [reply frontend/src/addPlace.js]: ooh i thought so users could add multiple places without manually refreshing the page, but i can remove it!

Comment (tylerhou) [reply frontend/src/addPlace.js]: That's a good reason; you should keep then.

File: frontend/src/addPlace.js (hunk 1 above)
Comment (sophia-qin) [reply frontend/src/addPlace.js]: edited and self-closed!

**Pull request merged**